OpenEdXMobile/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -158,6 +160,16 @@ public static CharSequence getRelativeTimeSpanString(@NonNull Context context, l
         }
     }
 
+    /**
+     * Method to check if the text only contains the paragraph tags<p></p> or not
+     */
```

Comment:
IMO `Method to check if the text only contains the HTML paragraph tag.` is more appropriate.
also add `@param` and `@return` with the doc string.